scenarios/remote_call/README.md

```diff
@@ -0,0 +1,15 @@
+## remote_call
+
+### Description
+
+This scenario tests the throughput of `remote_call` operations.
+
+**warning** This is a TryCP-based scenario and needs to be run differently to other scenarios.
+
+### Suggested command
+
+You can run the scenario locally with the following command:
+
+```bash
+RUST_LOG=info cargo run --package remote_call -- --targets targets.yaml --duration 300
```

Comment:
It doesn't work yet, is that what you meant that it's not complete?

```sh

Running `target/debug/remote_call --targets targets.yaml --duration 300`
#RunId: [TkU1tuVPmBLQF1N3mthta]
[2024-06-04T20:33:40Z INFO  wind_tunnel_runner::run] Running scenario: remote_call
⠁ [---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] [00:00:00 / 00:05:00][2024-06-04T20:33:40Z ERROR wind_tunnel_runner::run] Agent setup failed for agent agent-0: IO error: Connection refused (os error 61)
    
    Caused by:
        Connection refused (os error 61)
⠙ [#>-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] [00:00:02 / 00:05:00][2024-06-04T20:33:42Z ERROR wind_tunnel_runner::run] Agent setup failed for agent agent-1: IO error: No route to host (os error 65)
    
    Caused by:
        No route to host (os error 65)
[2024-06-04T20:33:42Z WARN  wind_tunnel_core::shutdown] Failed to send shutdown signal: SendError(())

Summary of operations
┌──────────────┬─────────────┬─────────────┬─────────────┬──────────────────┬───────────────────┐
│ operation_id │ avg_time_ms │ min_time_ms │ max_time_ms │ total_operations │ total_duration_ms │
└──────────────┴─────────────┴─────────────┴─────────────┴──────────────────┴───────────────────┘`
```

Reply:
@jost-s - did you have trycp servers running at the addresses specified in the `targets.yaml` file?

Reply:
I did not. That could be added to this readme.

Reply:
Is this what's supposed to happen?

```sh
Running `target/debug/remote_call --targets targets.yaml --duration 300`
#RunId: [sb4yG7VP6jLxTKEDjS8vw]
[2024-06-04T20:55:34Z INFO  wind_tunnel_runner::run] Running scenario: remote_call
⠤ [###########################>-----------------------------------------------------------------------------------] [00:01:14 / 00:05:00][2024-06-04T20:56:49Z ERROR wind_tunnel_runner::run] Agent setup failed for agent agent-1: IO error: Operation timed out (os error 60)
    
    Caused by:
        Operation timed out (os error 60)
⠲ [##############################################################################################################>] [00:04:59 / 00:05:00][2024-06-04T21:00:34Z WARN  wind_tunnel_core::shutdown] Failed to send shutdown signal: SendError(())

Summary of operations
┌────────────────────────┬─────────────┬─────────────┬─────────────┬──────────────────┬───────────────────┐
│ operation_id           │ avg_time_ms │ min_time_ms │ max_time_ms │ total_operations │ total_duration_ms │
├────────────────────────┼─────────────┼─────────────┼─────────────┼──────────────────┼───────────────────┤
│ trycp_configure_player │ 533.44      │ 508.32      │ 699.98      │ 561              │ 299259.78         │
├────────────────────────┼─────────────┼─────────────┼─────────────┼──────────────────┼───────────────────┤
│ trycp_reset            │ 0.41        │ 0.31        │ 1.06        │ 562              │ 230.56            │
└────────────────────────┴─────────────┴─────────────┴─────────────┴──────────────────┴───────────────────┘
```

Reply:
Looks right to me :+1:

Reply:
Yep, that looks great.

The local IP in the target.yaml is on my network so yes it won't work for anybody else. I intend to replace the contents of this list with HP addresses.

Once that's done though, it will still be normal for some addresses to be unavailable. So you will see the same error that you're seeing here. That shouldn't and doesn't terminate the scenario, just the one agent stops.